### DISCUSSION TITLE
Set a default property with the Animate tool

### DISCUSSION THREAD

**Discussion by ghost** [opentoonz/opentoonz#6354] [Ideas]
### Is your feature request related to a problem? Please describe

Currently, the Animate tool opens with the default position property, certainly the obvious choice. But it would be nice to be able to choose a default property like all or scale, for example. 

### Describe the solution you'd like

Right-click on a property and choose "set as default"

![set a defaut property_animate tool](https://github.com/opentoonz/opentoonz/assets/29790512/f15ee39b-3d26-421d-ae8b-312ed623b417)

**Comment by DarrenTAnims** [comment]
This would be really useful. I use "ALL" almost all of the time, so it would be nice to have this selected by default.
Rather than a context menu option that would be hard for new users to stumble across, adding it to the Animation section in the Preferences would be more consistent.

**Comment by beeheemooth** [comment]
You can assign the shortcut to the desired mode for the Animate Tool, or place it in the Command Bar (I did).
